.typos.toml

```diff
@@ -7,6 +7,7 @@ extend-exclude = [
     "tests/unit/materializers/test_built_in_materializer.py",
     "tests/integration/functional/cli/test_pipeline.py",
     "src/zenml/zen_server/dashboard/",
+    "examples/llm_finetuning/lit_gpt/"
```

Comment:
Why do you need to exclude the new template from the typos check? Can the typos be fixed on the template side instead?

Reply:
The template contains third-party code which I tried not to modify, as we'll have to swap that from time to time to upgrade it.